charts/jenkins/Chart.yaml

```
@@ -1,28 +1,28 @@
-apiVersion: v2
-appVersion: 2.462.3
-version: 1.0.1
-name: jenkins-by-lp
-description: "The open-source automation server that enables developers to build, test, and deploy their CI/CD software efficiently. Preconfigured by LinuxPolska"
-type: application
-annotations:
-  catalog.cattle.io/display-name: Jenkins
-  catalog.cattle.io/os: linux
-  catalog.cattle.io/permits-os: linux
-  catalog.cattle.io/release-name: jenkins-by-lp
-home: https://linuxpolska.com
-icon: https://linuxpolska.com/logo/LinuxPolska-icon.png
-keywords:
-- cicd
-sources:
-- https://github.com/SourceMation/charts.git
-dependencies:
-- name: common
-  repository: https://sourcemation.github.io/charts/
-  version: 0.1.x
-- name: jenkins
-  repository: https://charts.jenkins.io
-  version: 5.7.8
-kubeVersion: '>=1.26.0-0 <1.31.0-0'
-maintainers:
-- email: jaroslaw.mazurkiewicz@linuxpolska.com
-  name: Jarosław Mazurkiewicz
+apiVersion: v2
+appVersion: 2.479.3
+version: 1.0.2
+name: jenkins-by-lp
```

name according to example-chart
```suggestion
name: jenkins
```